vehicle/seat/cupra/provider.go

```diff
@@ -84,6 +84,18 @@ func (v *Provider) Range() (int64, error) {
 	return int64(res.Engines.Primary.Range.Value), err
 }
 
+var _ api.VehicleOdometer = (*Provider)(nil)
+
+// Odometer implements the api.VehicleOdometer interface
+func (v *Provider) Odometer() (float64, error) {
+	res, err := v.statusG()
+	if err == nil {
```

Comment:
please check how range does the simpler return. seems I cannot post to your repo to fix this.